### DISCUSSION TITLE
Linux/Arm 64-bit (`linux/arm64/v8`) release assets

### DISCUSSION THREAD

**Discussion by benz0li** [neovim/neovim#31153] [Show and tell]
*Unofficial* and *untested* (and *`strip`ped*) Linux/Arm 64-bit (`linux/arm64/v8`) release assets are available at https://gitlab.b-data.ch/neovim/neovim/-/releases.

No intention to provide AppImages.

**Comment by benz0li** [comment]
Multi-arch (`linux/amd64`, `linux/arm64/v8`) docker images: https://gitlab.b-data.ch/neovim/nvsi/container_registry.

Usage:

```Dockerfile
FROM glcr.b-data.ch/neovim/nvsi:nightly AS nvsi

FROM debian:12

COPY --from=nvsi /usr/local /usr/local
```

👉 `neovim/nvsi` images simply store a Neovim installation at `/usr/local` – you cannot use these to run `nvim` itself.

**Comment by adamency** [comment]
FYI, Neovim already provides builds for ARM64, at least since 0.10.4. (see #32059)

However would you mind providing linux arm64 builds for 0.9.x and 0.8.x if you can ?

**Reply by zeertzjq** [reply]
No, it's #32194

**Reply by adamency** [reply]
@zeertzjq thanks for the correction

**Reply by clason** [reply]
We will not be providing new builds for unsupported releases. You can build them yourself if you really need them.
